Client/Frontend/History/HistoryViewController.swift

```diff
@@ -71,6 +71,22 @@ class HistoryViewController: UITableViewController, UrlViewController {
         }
     }
 
+    func createSizedFavicon(icon: UIImage?) -> UIImage? {
```

Comment:
Can this be `func createSizedFavicon(icon: UIImage) -> UIImage` instead? So that it is the callers responsibility to pass in a valid favicon and make decision about what to do when it does not have a favicon?